src/content/app/tools/blast/types/blastApiResponse.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export type BlastApiResponse = {
+  result: {
+    hits: Hit[];
+  };
+};
+
+type Hit = HSP[];
+
+type HSP = {
```

Comment:
I remember Steve didn't like that it's called an hsp — a high-scoring segment pair — in the api. Is it an incorrect name for the data represented by this part of the response, @ens-st3 ? Is there a better term we should use instead?

Reply:
Did we find out from Steve whether he is okay with this or not?

Reply:
He says it's not a blocker for merging. I think the idea was to ask web-prod why they are calling these things HSPs, although the alignments in them contain gaps, whereas according to [NCBI definition](https://www.ncbi.nlm.nih.gov/books/NBK62051/) (and [Ensembl's definition](https://www.ensembl.org/Help/View?id=451) as well), an HSP does not contain gaps.

Reply:
Agreed